openapi-v1.yaml

```diff
@@ -6385,46 +6267,67 @@ paths:
+          required: false
+          type: string
+        - name: orderby
+          in: query
+          description: sort by which field valid values include last_accessed, size, name
+          required: false
+          type: string
+        - name: tag
+          in: query
+          description: tag to search for, more than one can be included
+          required: false
+          type: array
+          collectionFormat: multi
+          items:
+            type: string
+        - name: exclude_tag
+          in: query
+          description: tags to exclude matching array in results, more than one can be included
+          required: false
+          type: array
+          collectionFormat: multi
+          items:
+            type: string
+        - name: file_type
+          in: query
+          description: file_type to search for, more than one can be included
+          required: false
+          type: array
+          collectionFormat: multi
+          items:
+            type: string
+        - name: exclude_file_type
+          in: query
+          description: file_type to exclude matching array in results, more than one can be included
+          required: false
+          type: array
+          collectionFormat: multi
+          items:
+            type: string
+        - name: file_property
+          in: query
+          description: file_property key-value pair (comma separated, i.e. key,value) to search for, more than one can be included
+          required: false
+          type: array
+          collectionFormat: multi
+          items:
+            type: string
       responses:
         200:
           description: the group contents
           schema:
-            $ref: '#/definitions/GroupListing'
-        default:
-          description: error response
-          schema:
-            $ref: "#/definitions/Error"
-    patch:
-      description: Changes attributes of the group
-      operationId: updateGroup
-      tags:
-        - groups
-      parameters:
-        - in: body
-          name: GroupUpdate
-          schema:
-            $ref: '#/definitions/GroupUpdate'
-      responses:
-        204:
-          description: attributes changed successfully
-        default:
-          description: error response
-          schema:
-            $ref: "#/definitions/Error"
-    delete:
-      description: Deletes the group and all the subgroups recursively. The assets are not deleted nor are not relocated to any other group
-      operationId: deleteGroup
-      tags:
-        - groups
-      responses:
-        204:
-          description: group deleted successfully
+            $ref: "#/definitions/ArrayBrowserData"
         default:
           description: error response
           schema:
             $ref: "#/definitions/Error"
 
-  /groups/{namespace}/{name}/{asset_namespace}/{asset_name}:
+  /groups/{namespace}/{name}/{asset_namespace}/{array}:
```

Comment:
I think that this will be used to add groups in groups, so `asset_name` is a better fit.